src/InteractionModWork.H

```diff
@@ -63,6 +64,9 @@ class InteractionModWork : public InteractionModel<AC,ACT,ACTD,A>
         /*! \brief Simulate agent interaction at work */
         virtual void interactAgents( AC&, MultiFab& ) override;
 
+        /*! \brief Simulate agent interaction at work */
+        virtual void interactAgents( AC&, MultiFab&, AC& ) override {}
```

Comment:
In the non-travel interaction models, do you want to this function be
```
{
    amrex::Abort("Do not use this interface for this interaction model");
}

```
instead of a blank function?